Foundation/FileManager.swift

```diff
@@ -120,18 +122,333 @@ open class FileManager : NSObject {
         return result
     }
     
+    private enum _SearchPathDomain {
+        case system
+        case local
+        case network
+        case user
+        
+        init?(_ domainMask: SearchPathDomainMask) {
+            if domainMask == .systemDomainMask {
```

Comment:
1. domainMask is an OptionSet afaik, and urls function will return an array with multiple items if multiple masks are passes I think. I’m not sure however.

Reply:
No, they're documented to accept a single mask at a time. Weirdness of the Darwin API, but yeah.

Reply:
My bad — you're correct, and this patch is wrong. (_url…_ accepts a single mask, but _urls…_ does not.)